2.segmentation/run_segmentation.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "Running segmentation..."
+
+conda activate op_sam_env
+
+jupyter nbconvert --to=script --FilesWriter.build_directory=scripts notebooks/*.ipynb
+
+cd scripts
+
+python 0.segment_images.py
```

Comment:
```suggestion
python scripts/0.segment_images.py
```
To save a few lines of code, recommend removing the `cd scripts` and just including the path in the python statement.